src/main/java/com/rybka/command/Invoker.java

```diff
@@ -0,0 +1,12 @@
+package com.rybka.command;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class Invoker {
```

Comment:
Can't find this requirement in the task description. Point me please where you find that service to invoke command should be implemented. 🥇

Reply:
fixed